### PR TITLE
exclude newspix from getty

### PR DESCRIPTION
**Pull request by jamesgorrie**
We don't have a contract with them.

### CODE REVIEW

File: media-api/app/lib/elasticsearch/SearchFilters.scala [hunk 1]
```diff
@@ -18,7 +18,7 @@ trait SearchFilters extends ImageFields {
   val invalidFilter = Config.requiredMetadata.map(metadataField).toNel.map(filters.anyMissing)
 
   // New Cost Model
-  import UsageRightsConfig.{ suppliersCollectionExcl, freeSuppliers, payGettySourceList }
+  import UsageRightsConfig.{ suppliersCollectionExcl, freeSuppliers }
```
Comment (tsop14) [inline media-api/app/lib/elasticsearch/SearchFilters.scala]: Why did you remove 'payGettySourceList'?

Comment (jamesgorrie) [inline media-api/app/lib/elasticsearch/SearchFilters.scala]: It wasn't used.

**Comment by theefer** [comment]
:+1: but please let's check if we're out of sync with the source list of excluded collections, or if the source list is wrong.

**Comment by jamesgorrie** [comment]
I was wrong - it's newpix international.
They annoyingly* attached getty data then email pictures in.

* I say annoying, they actually attach composition data etc which is interesting, annoying that they do it using Getty's XMP

**Comment by theefer** [comment]
What's the best solution in that case?  Should we still exclude them from the Getty parser for the time being?

**Comment by jamesgorrie** [comment]
That's what I am looking at now - I wonder if other might use it too so when they upload to Getty (?) it get's categorised the way they want it.

**Comment by jamesgorrie** [comment]
#1395
